### ISSUE TITLE
assembly2 crashing freecad when adding or updating mor than 2 parts

### ISSUE THREAD

**Issue by linden1910** [hamish2014/FreeCAD_assembly2#62]
After updating to this freecad version:

OS: Linux Mint 17 Qiana
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.16.5805 (Git)
Build type: None
Branch: master
Hash: fe7c3cda444b00c53fa0a71333f40a3774278698
Python version: 2.7.6
Qt version: 4.8.6
Coin version: 4.0.0a
OCC version: 6.8.0.oce-0.17

I can only add or update one part to an assembly on the adding of the second part I get this error in the terminal that I used to start freecad with for trubleshooting. 

*** Abort *** an exception was raised, but no catch was found.
	... The exception is:SIGSEGV 'segmentation violation' detected. Address 58

Usaly it is adress 0 or 36 dose not seem to be any ryme or resone to the adress given. I have figered out if I add or update onepart in the assembly save and restart freecad I can ad a second part but this is very tiediose way to add multiple parts.

Is any one else experiancing this or have any sugestions or recomondations to prevent this.

Thanks Linden

**Comment by hamish2014** [comment]
Yes, see https://github.com/hamish2014/FreeCAD_assembly2/issues/59
